Makefile

```diff
@@ -186,12 +205,46 @@ docker-build-osm-preinstall:
 docker-build-osm-healthcheck:
 	docker buildx build --builder osm --platform=$(DOCKER_BUILDX_PLATFORM) -o $(DOCKER_BUILDX_OUTPUT) -t $(CTR_REGISTRY)/osm-healthcheck:$(CTR_TAG) -f dockerfiles/Dockerfile.osm-healthcheck --build-arg GO_VERSION=$(DOCKER_GO_VERSION) --build-arg LDFLAGS=$(LDFLAGS) .
 
+.PHONY: docker-build-init-fips
+docker-build-init-fips:
```

Comment:
In general, I would much prefer to parameterize what we need to in the existing builds to make it possible to build FIPS-compliant images without needing to add new completely separate codepaths for them. It seems like the main things that are different are a couple things in the source and then most of how the images are built, but I would like to keep from having those kinds of changes from bubbling all the way up into places like the release guide.

Reply:
+1